AUTHORS

```diff
@@ -1,5 +1,11 @@
-# Below is a list of people and organizations that have contributed
-# to the Dart www site. Names should be added to the list like so:
+# This is a list of some of the Dart website's contributors.
+#
+# This does not necessarily list everyone who has contributed code,
+# especially since many employees of one corporation may be contributing.
```

Comment:
In fact, it definitely doesn't list everyone who's contributed. :) So maybe just delete this sentence?

```suggestion
```